src/Akka.Persistence.Sql.Linq2Db/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -199,13 +194,33 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                     .IsColumn().IsNullable(false)
                     .HasLength(64)
                     .IsPrimaryKey()
-                    .Member(r => r.JournalOrderingId).HasColumnName(tagColumns.OrderingId)
-                    .IsColumn().IsPrimaryKey();
+                    .Member(r => r.SequenceNumber).HasColumnName(tagColumns.SequenceNumber)
+                    .IsColumn().IsNullable(false)
+                    .Member(r => r.PersistenceId).HasColumnName(tagColumns.PersistenceId)
+                    .HasLength(255)
+                    .IsColumn().IsNullable(false).IsPrimaryKey();
                 
                 if (config.ProviderName.ToLower().Contains("sqlite"))
```

Comment:
Why is the branch needed for Sqlite?

Reply:
Its to prevent linq2db from storing the numbers in "REAL" floating point data type, explicitly state that we want the column to have "INTEGER" data type.